project/src/main/java/com/google/sps/servlets/FilteredEventServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.Date;
+import com.google.gson.Gson;
+import com.google.sps.data.Event;
+import com.google.sps.data.User;
+import com.google.sps.utilities.CommonUtils;
+import com.google.sps.utilities.SpannerTasks;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/get-filtered-events")
```

Comment:
Might be personal preference, but I'd rename this from "get" to maybe just "filtered-events" so that, if there does end up being a doPost ever, it's not misleading/requiring renaming everywhere

Reply:
@yicyao It seems you resolved the comment without leaving a reply here? If you discussed this offline, please still leave a reply here for future readers.

Reply:
done, I didn't think of a doPost, thanks